src/vellum/workflows/workflows/base.py

```diff
@@ -437,15 +460,23 @@ def run(
                     workflow_definition=self.__class__,
                 ),
             )
+            rejected_event._set_context(self._context)
+            return rejected_event
 
         if (
             last_event.name == "workflow.execution.rejected"
             or last_event.name == "workflow.execution.fulfilled"
             or last_event.name == "workflow.execution.paused"
         ):
+            # Auto-print monitoring URL if monitoring is enabled
+            self._print_monitoring_info(last_event)
```

Comment:
[oos] Instead of inline into the workflow itself, this could be added as a `examples/workflows` to show how you used VellumEmitter in action.

As I said above, I think the automatic enablement of Vellum Monitoring will come through a separate project